stable/metallb/templates/config.yaml

```diff
@@ -0,0 +1,18 @@
+{{- if (or .Values.config .Values.arpCIDR) }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: metallb-system
+  name: config
```

Comment:
Add standard labels and use the `fullname` template for names. Apply everywhere.